protostar/testing/test_case_runners/test_case_runner_factory.py

```diff
@@ -21,11 +21,10 @@ def make(self, test_case: TestCase) -> TestCaseRunner:
         mode = self._state.config.mode
 
         assert mode, "Test mode should be determined at this point."
-
```

Comment:
nitpick: this file behaviour is not changed at all here, consider reverting changes